### PR TITLE
Move tables_query action to new class pattern

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

This PR refactors the `tables_query` to align with the newly introduced class pattern. Recently, additional logic was integrated for the single agent implementation (see https://github.com/dust-tt/dust/pull/5220). This PR extends that functionality by adding new logic to the class, introducing the `deprecatedBuildSpecificationForSingleActionAgent` method.

## Risk

Worst case it breaks the tables query action.

### CODE REVIEW

File: front/lib/api/assistant/actions/tables_query.ts [hunk 1]
```diff
@@ -1,8 +1,5 @@
 import type {
```
Comment (flvndvd) [inline front/lib/api/assistant/actions/tables_query.ts:1]: Better review with hide whitespace (click [here](https://github.com/dust-tt/dust/pull/5315/files?diff=split&w=1)).

File: front/lib/api/assistant/actions/tables_query.ts [hunk 2]
```diff
@@ -156,291 +155,285 @@ async function tablesQueryActionSpecification({
...
-    ],
-    {
-      conversationId: conversation.sId,
-      workspaceId: conversation.owner.sId,
-      agentMessageId: agentMessage.sId,
-    }
-  );
+    // Creating action
+    const action = await AgentTablesQueryAction.create({
+      tablesQueryConfigurationId: actionConfiguration.sId,
```
Comment (flvndvd) [inline front/lib/api/assistant/actions/tables_query.ts:246]: 🚗 We were using the wrong `sId` here.

Comment (fontanierh) [reply front/lib/api/assistant/actions/tables_query.ts:246]: If we don't use it (and I don't think we do), we might actually want to drop the field.
Will investigate 👍

**Review by fontanierh — APPROVED** [review]
LGTM

File: front/lib/api/assistant/actions/tables_query.ts (hunk 2 above)
Comment (flvndvd) [reply front/lib/api/assistant/actions/tables_query.ts:246]: Keep me posted :)

**Pull request merged**